### PR TITLE
preloader

### PR DESCRIPTION
**Pull request by YaroslavNikiforov**

### CODE REVIEW

File: front-end/src/App.scss [hunk 1]
```diff
@@ -22,6 +22,18 @@
   color: #61dafb;
 }
 
+.App-preloader {
```
Comment (Thealoner) [inline front-end/src/App.scss]: I suggest we use BEM in .block__element--modifier format everywhere. BEM is mentioned as highly recommended in the project requirements spec.

Comment (YaroslavNikiforov) [reply front-end/src/App.scss]: These are temporary styles for the default component <App/>. For the Preloader component, all styles are written according to the methodology.

Comment (YaroslavNikiforov) [reply front-end/src/App.scss]: all styles for the App component will be rewritten

Comment (Thealoner) [reply front-end/src/App.scss]: Got it

**Pull request merged**